### PR TITLE
feat(MUBI): add presence

### LINKED ISSUE #6956: Mubi.com - Service Add Request
### Website name

Mubi

### Website URL

https://mubi.com

### Website logo

https://mubi.com/logo

### Prerequisites

- [X] It is a paid service
- [ ] It displays NSFW content
- [ ] It is region restricted

### Description

Mubi is a curated streaming service, which offers an ever-changing collection of hand-picked films, introducing one new film each day. Mubi is also a distributor and producer.

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

Resolves #6956
Added the mubi presence.
## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

**Review by thisanickname — APPROVED** [review]

**Review by myvault999 — APPROVED** [review]
perfect

**Review by myvault999 — COMMENTED** [review]
perfect

**Review by myvault999 — APPROVED** [review]

File: websites/M/MUBI/metadata.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.7",
+  "service": "MUBI",
+  "author": {
+    "id": "638080361179512853",
+    "name": "Dark_Ville#2756"
```
Comment (theusaf) [inline websites/M/MUBI/metadata.json:6]:
```suggestion
    "name": "Dark_Ville"
```

Comment (DarkVillager) [reply websites/M/MUBI/metadata.json:6]: @Timeraa fix pmd

**Review by Bas950 — APPROVED** [review]

File: websites/M/MUBI/metadata.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
+{
```
Comment (EncryptedDev) [inline websites/M/MUBI/metadata.json:1]: Not too sure why CI didn't pick up on this, but this hasn't been formatted correctly. Fix this by running lint

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/M/MUBI/metadata.json (hunk 2 above)
Comment (DarkVillager) [reply websites/M/MUBI/metadata.json:1]: ok

File: websites/M/MUBI/presence.ts [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+						.querySelector("__next-route-announcer__")
+						?.textContent?.split("|")[0] ??
+					document.querySelector("title")?.textContent?.split("|")[0];
+				if (video.duration && !video.paused)
+					presenceData.endTimestamp = presence.getTimestampsfromMedia(video)[1];
+				presenceData.largeImageKey =
+					document
+						.querySelector("#__NEXT_DATA__")
+						?.innerHTML?.match(
+							/https:\/\/images.mubicdn.net\/images\/(film|show)\/[0-9]*\/cache-[0-9]*-[0-9]*\/image-w1280[.]jpg/
```
Comment (theusaf) [inline websites/M/MUBI/presence.ts:158]: Regex consistency
```suggestion
							/https:\/\/images[.]mubicdn[.]net\/images\/(film|show)\/[0-9]*\/cache-[0-9]*-[0-9]*\/image-w1280[.]jpg/
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

File: websites/M/MUBI/metadata.json (hunk 2 above)
Comment (EncryptedDev) [reply websites/M/MUBI/metadata.json:1]: Wasn't fixed, but resolved. Great!

**Pull request merged**